coder_review.common.inc

```diff
@@ -1238,7 +1283,12 @@ function do_coder_review_callback(array &$coder_args, array $review, array $rule
   $function = $rule['#value'];
   if ($function) {
     if (function_exists($function)) {
-      call_user_func_array($function, array(&$coder_args, $review, $rule, $lines, &$results));
+      call_user_func_array($function, array(
+        &$coder_args,
+        $review, $rule,
```

Comment:
not sure if this entire array should be wrapping in multiple lines or not, but if so, these two elements should not be in the same line then.